packages/attendance/src/utils/functions/HandleGenderList.ts

```diff
@@ -0,0 +1,20 @@
+import { MALE, FEMALE } from "./Constants";
+
+export interface IHandleGenderList {
+  (students: any, t: any): number[];
+}
+
+// Returns number of students
+// which belong to the specified gender
+export const HandleGenderList: IHandleGenderList = (students, t) => {
```

Comment:
Follow method name conventions. Should it be handleGenderList ?
Also Function name Handle... does not looks correct. it is filter function so please correct function name